cluster-up/up.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -xe
```

Comment:
intentional to stay?
or a debug

Reply:
debug I have to remove, need not to hurry things up.